lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -5,32 +5,39 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 @use '../../scss-shared/nx-focus-border-helpers';
+@use '../../scss-shared/nx-radio-checkbox-helpers';
 
-$-height: 16px;
-$-width: 16px;
+$-size: 16px;
+$-checked-border-width: 4px;
 
 .nx-radio__circle {
-  height: $-height;
-  width: $-width;
+  background-color: var(--nx-color-component-background);
+  border-radius: 50%;
+  height: $-size;
+  width: $-size;
 }
 
-.nx-radio__inner-circle {
-  fill: var(--nx-swatch-white);
-  stroke: var(--nx-swatch-blue-40);
-}
+.nx-radio__input {
+  &:disabled {
+    + .nx-radio__circle {
+      @include nx-radio-checkbox-helpers.multi-inset-box-shadow($-checked-border-width);
 
-.nx-radio__input:checked {
-  + .nx-radio__circle .nx-radio__outer-circle {
-    fill: none;
+      background-color: var(--nx-color-radio-disabled-background);
+    }
   }
 
-  &:disabled {
-    + svg.nx-radio__circle {
-      stroke: var(--nx-color-interactive-border-disabled);
+  &:checked {
+    &, &:hover {
+      + .nx-radio__circle {
+        @include nx-radio-checkbox-helpers.inset-box-shadow(var(--nx-color-radio-checkbox-selected), $-checked-border-width);
+      }    
+    }
+
+    &:disabled {
+      + .nx-radio__circle {
+        @include nx-radio-checkbox-helpers.multi-inset-box-shadow($-checked-border-width);
```

Comment:
Can be slightly simplified similar to https://github.com/sonatype/sonatype-react-shared-components/pull/1048#discussion_r1163218108

Reply:
> Can be slightly simplified similar to [#1048 (comment)](https://github.com/sonatype/sonatype-react-shared-components/pull/1048#discussion_r1163218108)

This is simplified here: 43a72297fbdddfc7ef57ab1e73a1a3a37d033b1c